src/ngamsPlugIns/ngamsPlugIns/mwa/ngamsMWAOnlinePlugIn.py

```diff
@@ -180,4 +180,4 @@ def _restoreSubscriptionInfoFromDisk(srvObj):
     except Exception:
         logger.exception('Fail to append filelist to subscription info list')
     finally:
-        srvObj._subscriptionSem.release()
\ No newline at end of file
+        srvObj._subscriptionSem.release()
```

Comment:
Could this change be left out? I'm sure it was unintentional. There is a similar one in the other modified file.